content/blog/deploying-super-mario-game-on-kubernetes-in-hpe-greenlake-for-private-cloud-enterprise.md

```diff
@@ -0,0 +1,477 @@
+
+T﻿ype the following commend to check that all the game service endpoints have been populated:
+
+```shell
+$ kubectl get endpoints -n cfe-games
+NAME             ENDPOINTS                            AGE
+mario-service    10.192.3.118:80,10.192.4.32:80       60s
+tetris-service   10.192.3.119:3000,10.192.4.33:3000   50s
+```
+
+### Set up the load balancer with *MetalLB*
+
+You can install *MetalLB* and set up the load balancer in the K8s cluster by following up the blog post [Setting up the load balancer with MetalLB](https://developer.hpe.com/blog/set-up-load-balancer-with-metallb-in-hpe-greenlake-for-private-cloud-enterprise/).
+
+H﻿ere is the deployed *MetalLB* to the namespace *metallb-system* in the cluster:
+
+```shell
+$ kubectl get all -n metallb-system
+NAME                              READY   STATUS    RESTARTS   AGE
+pod/controller-57b4fdc957-dr4h4   1/1     Running   0          18d
+pod/speaker-9kx9h                 1/1     Running   0          18d
+pod/speaker-d6sdh                 1/1     Running   0          18d
+pod/speaker-gxbbx                 1/1     Running   0          18d
+pod/speaker-hflbj                 1/1     Running   0          18d
+pod/speaker-wfw9n                 1/1     Running   0          18d
+
+NAME                      TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)   AGE
+service/webhook-service   ClusterIP   10.107.242.167   <none>        443/TCP   18d
+
+NAME                     DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR            AGE
+daemonset.apps/speaker   5         5         5       5            5           kubernetes.io/os=linux   18d
+
+NAME                         READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/controller   1/1     1            1           18d
+
+NAME                                    DESIRED   CURRENT   READY   AGE
+replicaset.apps/controller-57b4fdc957   1         1         1       18d
+```
+
+By running the following commands, you can see a range of virtual IP addresses, *"10.6.115.251-10.6.115.254"*, defined in the CRD resource *IPAddressPool*, and the layer 2 service IP address announcement in the CRD resource *L2Advertisement*:
+
+```shell
+$ kubectl get ipaddresspools -n metallb-system
+NAME       AUTO ASSIGN   AVOID BUGGY IPS   ADDRESSES
+cfe-pool   true          false             ["10.6.115.251-10.6.115.254"]
+
+
+
+$ kubectl get l2advertisements -n metallb-system
+NAME           IPADDRESSPOOLS   IPADDRESSPOOL SELECTORS   INTERFACES
+cfe-l2advert   ["cfe-pool"]
+```
+
+### Generate a self-signed certificate using cert-manager
+
+You can d﻿eploy cert-manager to the K8s cluster and generate a self-signed certificate by following up the blog post [Generating self-signed certificates using cert-manager](https://developer.hpe.com/blog/generating-self-signed-certificates-using-cert-manager-for-kubernetes-in-hpe-greenlake-for-private-cloud-entreprise/).
```

Comment:
Similarly, change "by following up the blog post" to:
"...certificate by following the instructions found in the blog post..."